$OpenBSD: patch-configure_in,v 1.2 2011/05/19 15:28:27 ajacoutot Exp $

- Remove over-the-top optimization
- Fix pthread library
- Remove gnome-vfs2 dependency.

--- configure.in.orig	Wed Dec  9 01:34:33 2009
+++ configure.in	Wed May 18 20:41:16 2011
@@ -45,8 +45,8 @@ dnl the library paths with strings that have longer le
 case "$host_os" in
 darwin*) 	CFLAGS="-g -O2 -Wall"
 		LDFLAGS="-headerpad_max_install_names";;
-*)		CFLAGS="-g -O6 -Wall"
-		CXXFLAGS="-g -O6 -Wall";;
+*)		CFLAGS="${CFLAGS} -Wall"
+		CXXFLAGS="${CXXFLAGS} -Wall";;
 esac
 
 if test $CC = "gcc"; then
@@ -92,7 +92,7 @@ AM_PATH_XMEDCON(0.10.0, FOUND_XMEDCON=yes, FOUND_XMEDC
 
 dnl switch to C++ for DCMTK library stuff - also, if pthread is on the platform, probably need that
 AC_LANG_CPLUSPLUS
-AC_CHECK_LIB(pthread, pthread_mutex_init, THREAD_LIBS="-lpthread", THREAD_LIBS="")
+AC_CHECK_LIB(pthread, pthread_mutex_init, THREAD_LIBS="-pthread", THREAD_LIBS="")
 AC_CHECK_LIB(dcmdata, main, FOUND_DCMDATA=yes, FOUND_DCMDATA=no, -lofstd  -lz -L/usr/local/dicom/lib -L/sw/lib $THREAD_LIBS)
 
 dnl trying to phase out libfame use in favor of ffmpeg
@@ -286,7 +286,6 @@ PKG_CHECK_MODULES(AMIDE_GTK,[
 ## gconf stuff is encapsulated in amide_gconf.c
 if (test $native_win32 = no); then
    PKG_CHECK_MODULES(AMIDE_GTK_EXTRA,[
-		gnome-vfs-2.0   >= 2.16.0
 		gconf-2.0	>= 2.14.0
    ])
    AMIDE_GTK_LIBS="$AMIDE_GTK_LIBS $AMIDE_GTK_EXTRA_LIBS"
